### PR TITLE
06 add manifest readme

### PR DESCRIPTION
**Pull request by duncan-ross**

### CODE REVIEW

File: viz/animated-bar/src/manifest.json [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-  "termsOfServiceUrl": "https://url",
-  "packageUrl": "https://url",
+  "name": "Animated Bar Chart",
+  "organization": "Duncan Ross",
+  "description": "Animated bar chart race for unique data visualization",
+  "logoUrl": "https://raw.githubusercontent.com/googledatastudio/experimental-visualizations/master/viz/animated-bar/resources/icon.png",
+  "organizationUrl": "https://datastudio.google.com",
+  "supportUrl": "https://github.com/googledatastudio/experimental-visualizations/issues",
+  "privacyPolicyUrl": "https://docs.google.com/document/d/1RJXX3rAG29YBxQJIfrp887jGoRg3qLEtnY501BiVmIU/edit?usp=sharing",
+  "termsOfServiceUrl": "https://docs.google.com/document/d/1RJXX3rAG29YBxQJIfrp887jGoRg3qLEtnY501BiVmIU/edit?usp=sharing",
```
Comment (bjg96) [inline viz/animated-bar/src/manifest.json:9]: You'll need to make that doc available to all via link. I can't access it currently.

Comment (duncan-ross) [reply viz/animated-bar/src/manifest.json:9]: Link should be public. None of my docs can be shared/accessed by external accounts due to my @google acct restrictions, so I may need to generate the doc on my personal acct.

Comment (duncan-ross) [reply viz/animated-bar/src/manifest.json:9]: https://docs.google.com/document/d/1b80nGcl4VlRDLeE8FIajLmgDh3knidwAAOVu93n_bic/edit?usp=sharing

Comment (bjg96) [reply viz/animated-bar/src/manifest.json]: Great, thanks!

**Pull request merged**